crmsh/sbd.py

```diff
@@ -469,3 +473,19 @@ def get_sbd_value_from_config(key):
         conf = utils.parse_sysconfig(SYSCONFIG_SBD)
         res = conf.get(key)
         return res
+
+    def adjust_pcmk_delay_max(self):
```

Comment:
Currently, adjust_pcmk_delay_max() only deal with disk-based sbd and diskless sbd. 

In theory it applies to other non-sbd stonith device types too, for 2-node cluster without qdevice. for example. 

crm configure show type:primitive | grep -e stonith:

And, adjust_pcmk_delay_max() can be called anywhere, even outside of bootstrap. With that, I prefer to move this function to cibconfig.py, 

Moreover, there are more complex situations with multiple stonith devices in the system. Maybe we only start with  the scope for the cluster with one stonith device, for the time being. And, this is good to ask Yan's advice too.